### PR TITLE
Feature/remove bitcoin core dependency

### PR DESCRIPTION
**Pull request by Yamaguchi**
This PR removes the dependency for "bitcoin-core" library (https://www.npmjs.com/package/bitcoin-core) and their configuration.

This PR includes changes to use esplora REST API (https://github.com/chaintope/esplora-tapyrus/blob/main/doc/API.md) instead of RPC call to tapyrusd and Electrum protocol.

### CODE REVIEW

**Review by azuchi — CHANGES_REQUESTED** [review]
In setup.md: 
> Consists of backend (calling Tapyrusd & Electrs-Tapyrus to get data) and frontend

should be changed.

File: backend/test/libs/util.spec.js [hunk 1]
```diff
@@ -7,6 +7,27 @@ describe('util', () => {
...
+      assert.strictEqual(util.isColorId(nft), true);
+    });
+
+    it('should return false if it is invalid', () => {
+      const invalidColorId =
+        '00FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF';
+      assert.strictEqual(util.isColorId(invalidColorId), false);
+    });
+  });
+
```
Comment (azuchi) [inline backend/test/libs/util.spec.js:30]: It would be better to have a test with over or under data length, data other than hex values.

**Review by azuchi — COMMENTED** [review]
`depends_on` definition of backedn service in `docker-compose.yml`,  still necessary `tapyrusd`?

File: backend/test/libs/util.spec.js (hunk 1 above)
Comment (Yamaguchi) [reply backend/test/libs/util.spec.js:42]: Add test case (a024c55)

**Review by azuchi — APPROVED** [review]

**Pull request merged**